### ISSUE TITLE
Support deleting / replacing text in text of inline forwarded mail

### ISSUE THREAD

**Issue by RealRaven2000** [RealRaven2000/SmartTemplates#172]
Expand the functions `%deleteQuotedText()%` and `%replaceQuotedText()%` to be used on the body of a forwarded Email. This should work when the template is applied while forwarding an Email inline. To show that the quoted text should be affected and make the change backwards compatible, the _quoteLevel_ parameter should be explicitly set to 0

Should also be able to support the functions `%deleteQuotedTags()%` and `%replaceQuotedTags()%`.

Usage examples:

```
%deleteQuotedText("multiple",0)%
%replaceQuotedText("Daily","nightly",0)%
%deleteQuotedTags("div.helter",0)%
%replaceQuotedTags("div.skelter","<p>you're welcome!</p>",0)%
```

**Comment by RealRaven2000** [comment]
Here is a test version:

[smartTemplate-fx-3.10.2pre42.zip](https://github.com/RealRaven2000/SmartTemplates/files/8111112/smartTemplate-fx-3.10.2pre42.zip)

To install, download the zip file and then drag it into Thunderbird Add-ons manager (don't unpack contents).

**Comment by RealRaven2000** [comment]
Implemented in v 3.11 published on [18/03/2022](https://addons.thunderbird.net/thunderbird/addon/324497/versions/3.11)
